test/Parse/ConditionalCompilation/language_version_explicit.swift

```diff
@@ -28,3 +28,30 @@
   asdf asdf asdf asdf
 #endif
 
+#if swift(<5)
+  let a = 1
```

Comment:
To future-proof these tests, I think the conditions should be such that `asdf asdf asdf asdf` emits no diagnostics in an `#if` branch which is never executed.